substrate/primitives/api/test/tests/ui/deprecation_info.stderr

```diff
@@ -0,0 +1,44 @@
+error: Invalid deprecation attribute: missing `note`
+  --> tests/ui/deprecation_info.rs:20:3
+   |
+20 |         #[deprecated(unknown_kw = "test")]
+   |         ^
+
+error: Invalid deprecation attribute
+  --> tests/ui/deprecation_info.rs:22:3
+   |
+22 |         #[deprecated(since = 5)]
+   |         ^
+
+error: Invalid deprecation attribute
+  --> tests/ui/deprecation_info.rs:24:3
+   |
+24 |         #[deprecated = 5]
+   |         ^
+
```

Comment:
as suggested those could give more information as they are the first to be show up to user.

Reply:
pushed the suggested change